vsphere/datadog_checks/vsphere/resource_filters.py

```diff
@@ -0,0 +1,92 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from typing import List, Pattern, Tuple
+
+from pyVmomi import vim
+
+from datadog_checks.vsphere.types import InfrastructureData
+
+
+def make_inventory_path(mor, infrastructure_data):
+    # type: (vim.ManagedEntity, InfrastructureData) -> str
+    mor_name = infrastructure_data[mor].get('name', '')
+    mor_parent = infrastructure_data[mor].get('parent')
+    if mor_parent:
+        return make_inventory_path(mor_parent, infrastructure_data) + '/' + mor_name
+    return ''
+
+
+def match_any_regex(string, regexes):
+    # type: (str, List[Pattern]) -> bool
+    for regex in regexes:
+        match = regex.match(string)
+        if match:
+            return True
```

Comment:
(nit)
```suggestion
        return bool(regex.match(string))
```

Reply:
No, this will return on the first iteration no matter what

Reply:
oh, yes 🤦‍♂